include/zephyr/lorawan/lorawan.h

```diff
@@ -217,6 +217,15 @@ typedef uint8_t (*lorawan_battery_level_cb_t)(void);
  */
 typedef void (*lorawan_dr_changed_cb_t)(enum lorawan_datarate dr);
 
+/**
+ * @brief Defines the descriptor callback handler function signature.
+ *
+ * @param descriptor Descriptor field
```

Comment:
Where does this value come from, what does it mean, and how should the user use the value?
This is a function the user needs to implement, so the docs need to explain what the expected inputs & outputs are.